### PR TITLE
evesty76767 saidsii dthisi woruosdl get meghrteghd

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: iiaimgm orgofokabal
add: AMMKKE SMYSDEWFL QAMDIN ON SGEGRBV T STATRITYUOP???
add:  FDJFJ???
/:cl:

asiiis ihe right?

### CODE REVIEW

**Comment by X-TheDark** [comment]
"How to get banned from GitHub, the guide"

**Comment by evsey9** [comment]
THE ABSOLUTE M A D M A N

If you fix this so it actually admins you, i will praise you  and give you 20$

**Comment by optimumtact** [comment]
Sure goof we'll merge this in the morning

**Comment by Iamgoofball** [comment]
tyahhhananalkk uuiuiuuuuuuu

File: code/world.dm [hunk 1]
```diff
@@ -10,6 +10,7 @@ var/global/list/map_transition_config = MAP_TRANSITION_CONFIG
 
 /world/New()
 	map_ready = 1
+	mmkakkedghofg ooofball adnan aDSMIJHN("iamgoofball", true)
```
Comment (evsey9) [inline code/world.dm]: `admin("iamgoofball", true)`

**Comment by evsey9** [comment]
isn't it 'admin("iamgoofball", true)` ?
i don't know BYOND coding at all to say anything, tho

**Comment by evsey9** [comment]
also, someone please teach me how to make pull requests

i want to fix this so it works

**Comment by evsey9** [comment]
time for me to learn byond coding and fix this branch

File: code/world.dm (hunk 1 above)
Comment (xxalpha) [inline code/world.dm]: ``true`` needs to be ``TRUE``

**Comment by evsey9** [comment]
me cris

**Pull request closed without merging**